File: 04-CSHARP/04-Condicionales/03ejercicioDiaSemana.cs
// Ejercicio dia de la semana en C#

int diaSemana;
Console.WriteLine("Ingresa un dia de la semana 1 - 5");
diaSemana = Convert.ToInt32(Console.ReadLine());

if (diaSemana == 1)
    Console.WriteLine("Es lunes");
else if (diaSemana == 2)
    Console.WriteLine("Es martes");
else if (diaSemana == 3)
    Console.WriteLine("Es miercoles");
else
    Console.WriteLine("Valor erroneo");

File: 03_CPP/04_condicionales/03ejercicioDiaSemana.cpp
// Ejercicio de dia de la semana en C++
#include <iostream>
using namespace std;

int main()
{

    int diaSemana;
    cout << "Ingresa un dia de la semana (1 - 3) " << endl;
    cin >> diaSemana;

    // Validar el dia de la semana
    if (diaSemana == 1)
        cout << "Hoy es Lunes" << endl;
    else if (diaSemana == 2)
        cout << "Hoy es martes" << endl;
    else if (diaSemana == 3)
        cout << "Hoy es miercoles" << endl;
    else
        cout << "Valor invalido" << endl;

    return 0;
}